docs/en/03_apache-solr.md

```diff
@@ -0,0 +1,71 @@
+# Apache solr
+
+Apache Solr is a fulltext search engine, an aspect which is often used
+alongside this module. But more importantly for us, it has bindings to [Apache Tika](http://tika.apache.org/)
+through the [ExtractingRequestHandler](http://wiki.apache.org/solr/ExtractingRequestHandler) interface.
+This allows Solr to inspect the contents of various file formats, such as Office documents and PDF files.
+The textextraction module retrieves the output of this service, rather than altering the index.
+With the raw text output, you can decide to store it in a database column for fulltext search
+in your database driver, or even pass it back to Solr as part of a full index update.
+
+In order to use Solr, you need to configure a URL for it (in `mysite/_config/config.yml`):
```

Comment:
```suggestion
In order to use Solr, you need to configure a URL for it (in `app/_config/config.yml`):
```
I'm gonna stop pointing these out - please update all `mysite` to `app`

Reply:
Done